sklearn/mixture/tests/test_mixture_base.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+
+from ...datasets.samples_generator import make_spd_matrix
+from ...utils.testing import assert_array_equal
+from ...utils.testing import assert_raise_message
+
+rng = np.random.RandomState(0)
```

Comment:
[it's really ugly :)](https://github.com/scikit-learn/scikit-learn/pull/6125#discussion-diff-50506337)